### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by shadowcan13** [Yisus7u7/termux-desktop-xfce#25] [General]
Pede senha para iniciar no termux mas eu não criei nem uma senha

**Comment by JesusChapman** [comment]
English or Spanish please

**Reply by shadowcan13** [reply]
English please

**Reply by JesusChapman** [reply]
You must create a password, it is normal when creating the vnc service

**Reply by shadowcan13** [reply]
> Estoy escrito en tu canal. Sería bueno que tomaras un tutorial para instalar la interfaz paso a paso.

The script already provides everything you need for installation

**Reply by shadowcan13** [reply]
> English please

Puede ser espanhol tambien

**Reply by JesusChapman** [reply]
Use English

**Reply by shadowcan13** [reply]
Ok I'll redo the process and take a screenshot of the problem I'm having

**Reply by shadowcan13** [reply]
your work is excellent thanks for the support

**Reply by JesusChapman** [reply]
Ok

**Reply by shadowcan13** [reply]
can you call me on whats app? 5531989298234

**Reply by shadowcan13** [reply]
Printscreen

**Reply by JesusChapman** [reply]
Hmm

**Reply by shadowcan13** [reply]
I need you help for its 😔

**Reply by JesusChapman** [reply]
https://t.me/offtopictermuxgroup

**Reply by shadowcan13** [reply]
I was excluded from the group affz
